types/react-stickynode/index.d.ts

```diff
@@ -40,35 +41,40 @@ declare namespace Sticky {
 
         /**
          * The offset from the top of window where the top of the element will
-         * be when sticky state is triggered(0 by default ).If it is a selector
-         * to a target(via `querySelector()`), the offset will be the height of
+         * be when sticky state is triggered(0 by default ). If it is a selector
+         * to a target (via `querySelector()`), the offset will be the height of
          * the target.
          */
         top?: number | string;
 
         /**
          * The offset from the top of document which release state will be
-         * triggered when the bottom of the element reaches at.If it is a
-         * selector to a target(via `querySelector()`), the offset will be the
+         * triggered when the bottom of the element reaches at. If it is a
+         * selector to a target (via `querySelector()`), the offset will be the
          * bottom of the target.
          */
         bottomBoundary?: number | string;
 
         /** z-index of the sticky */
         innerZ?: number | string;
 
-        /** Enable the use of CSS3 transforms (true by default). */
+        /** Enable the use of CSS3 transforms (`true` by default). */
         enableTransforms?: boolean;
 
         /**
          * Class name to be applied to the element when the sticky state is
-         * active (active by default).
+         * active ('active' by default).
          */
         activeClass?: string;
 
+        /**
+         * 	Class name to be applied to the inner element ('' by default).
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/react-stickynode/index.d.ts;
  Line: 71;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.